paddle/operators/CMakeLists.txt

```diff
@@ -46,20 +46,30 @@ endfunction()
 
 add_subdirectory(math)
 
-list(REMOVE_ITEM GENERAL_OPS
-     net_op
-     minus_op
-     mul_op
-     recurrent_op
-     scale_op)
+set(ONLYCPU_OPS
+    net_op
+    gather_op
+    scatter_op)
+foreach(src ${ONLYCPU_OPS})
+    op_library(${src} SRCS ${src}.cc)
```

Comment:
Does this make it simpler if we make `op_library` smarter, just call `op_library(${src})`, then it can do the following:

find if have both `.cc` and  `.cu` file, call `cc_library` and `nv_library` to add corresponding library.

So we can call `op_library(${src})` for general ops and `op_library(${src} DEPS xxx)` for ops have dependencies.

Reply:
You are right, I will refine it.

Most ops have the same name for `.cc` and `.cu`. But for `recurrent_op`, the `${src}` is not `recurrent_op.cc` and `recurrent_op.cu`.

Thus, `if ${src} is none`, cmake will auto find `${src}.cc` and `${src}.cu`. If user specific `${src}`,  cmake will use what user specify.